packages/components/src/core/Tag/index.stories.tsx

```diff
@@ -102,102 +111,245 @@ export const Default = {
-          {...props}
-          color={color}
-          icon={undefined}
-          label={label}
-          sdsStyle="rounded"
-          sdsType="primary"
-        />
+  // loop through all ICON_OPTIONS + create headers for SDS_TYPES
+  function TagType({
+    sdsStyle,
+    color,
+    sdsType,
+  }: {
+    sdsStyle: ExtraTagProps["sdsStyle"];
+    color: ExtraTagProps["tagColor"];
+    sdsType: ExtraTagProps["sdsType"];
+  }) {
+    return (
+      <div style={displayContents}>
+        <h4 style={h4Label}>
+          Type: <b>{sdsType}</b>
+        </h4>
+        {ICON_OPTIONS.map((icon) => {
+          return (
+            <TagIcon
+              sdsStyle={sdsStyle}
+              color={color}
+              sdsType={sdsType}
+              icon={icon}
+              key={String(icon)}
+            />
+          );
+        })}
       </div>
+    );
+  }
 
-      <div style={{ gridArea: "2 / 2 / 3 / 3" }}>
-        <RawTag
-          {...props}
-          color={color}
-          icon={undefined}
-          label={label}
-          sdsStyle="rounded"
-          sdsType="secondary"
-        />
+  // loop through all HOVERABLE_OPTIONS + create headers for ICON_OPTIONS
+  function TagIcon({
+    sdsStyle,
+    color,
+    sdsType,
+    icon,
+  }: {
+    sdsStyle: ExtraTagProps["sdsStyle"];
+    color: ExtraTagProps["tagColor"];
+    sdsType: ExtraTagProps["sdsType"];
+    icon: (typeof ICON_OPTIONS)[number];
+  }) {
+    return (
+      <div style={displayContents}>
+        <h5 style={h5Label}>
+          Icon: <b>{icon ? "yes" : "no"}</b>
+        </h5>
+        {HOVERABLE_OPTIONS.map((hover) => {
+          return (
+            <TagState
+              sdsStyle={sdsStyle}
+              color={color}
+              sdsType={sdsType}
+              icon={icon}
+              hover={hover}
+              key={String(hover)}
+            />
+          );
+        })}
       </div>
+    );
+  }
+
+  // loop through all PSEUDO_STATES + create headers for HOVERABLE_OPTIONS, PSEUDO_STATES
+  function TagState({
+    sdsStyle,
+    color,
+    sdsType,
+    icon,
+    hover,
+  }: {
+    sdsStyle: ExtraTagProps["sdsStyle"];
+    color: ExtraTagProps["tagColor"];
+    sdsType: ExtraTagProps["sdsType"];
+    icon: (typeof ICON_OPTIONS)[number];
+    hover: (typeof HOVERABLE_OPTIONS)[number];
+  }) {
+    const { label } = props;
 
-      <div style={{ gridArea: "2 / 3 / 3 / 4" }}>
-        <RawTag
-          {...props}
-          color={color}
-          icon={
-            icon || <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />
-          }
-          label={label}
-          sdsStyle="rounded"
-          sdsType="secondary"
-        />
+    return (
+      <div style={penultimateLevel}>
+        {PSEUDO_STATES.map((state) => {
+          return (
+            <div style={bottomLevel}>
+              {(hover === true || (hover === false && state === "default")) && (
+                <>
+                  <h6 style={bottomLabel}>
```

Comment:
Removed irrelevant `hover` iterations: when combined with all pseudo-states except default, `hover=false` is impossible